siem_integrations/clx_query_service/clxquery/views.py

```diff
@@ -0,0 +1,50 @@
+import re
+import os
+import logging
+
+from clxquery import utils
+from django.http import HttpResponse, JsonResponse
+from django.views.decorators.csrf import csrf_exempt
+from clx.io.reader.blazingsql_reader import BlazingSQLReader
```

Comment:
Are we still using the clx BlazingSQLReader? I don't see it in the branch. If so, we would have to add blazing as a dependency.

Reply:
I'm creating BlazingSQLReader instance for every query to release memory as soon as the request is processed, and also we are running it on multiple workers. Thought using  BlazingSQLReader would be a good idea. 
Yea, it would need your branch to be merged. I might not be correct, let me know what do you think?